sky/engine/platform/graphics/filters/FilterEffect.cpp

```diff
@@ -556,20 +556,20 @@ SkImageFilter::CropRect FilterEffect::getCropRect(const FloatSize& cropOffset) c
     if (hasX()) {
         rect.setX(boundaries.x());
         flags |= SkImageFilter::CropRect::kHasLeft_CropEdge;
-        flags |= SkImageFilter::CropRect::kHasRight_CropEdge;
+        flags |= SkImageFilter::CropRect::kHasWidth_CropEdge;
```

Comment:
Oh wait, not lgtm. The CropRect right is not equivalent to its width (and so on and so forth)